docs/user/vsphere/install_upi.md

```diff
@@ -255,6 +255,26 @@ TODO RHEL CoreOS does not have assets for vSphere.
 
 Set the vApp properties of the VM to set the Ignition config for the VM. The `guestinfo.ignition.config.data` property is the base64-encoded Ignition config. The `guestinfo.ignition.config.data.encoding` should be set to `base64`.
 
+### Configuring RHCOS VMs with static IP addresses
+
+As of OpenShift 4.6, you may use a new `guestinfo.afterburn.initrd.network-kargs` to configure static IP addressing.  
+The CoreOS "afterburn" component will lookup for the well-known key `guestinfo.afterburn.initrd.network-kargs` and use its value instead of the default.
```

Comment:
So this only sets the first boot and what about after that??

Should we show a complete example here?

Reply:
Hmm you're right, we may need to document here how to also embed the static addressing in the Ignition config to persist for subsequent boots.  I will get back to you on this.

Reply:
Hmm that is a good point what if I have to re-address my RHCOS virtual machine?

Reply:
Create a new machine, i.e. reprovision.

Reply:
This plugs into the same flow as the bare-metal: kargs should get translated into NM configuration in the initramfs, and that is forwarded to the real rootfs.